dali/base/dadfs.cpp

```diff
@@ -3689,6 +3690,9 @@ protected: friend class CDistributedFilePart;
         unsigned nc = fdesc->numClusters();
         if (nc) {
             for (unsigned i=0;i<nc;i++) {
+#if 1
+                IClusterInfo &cluster = OLINK(*fdesc->queryClusterNum(i));
```

Comment:
This was changed because a) I think it's equivalent to get the existing IClusterInfo from the IFileDescriptor it's processing, and more importantly in the context of this PR, the pre-existing code would involve group lookup, that called Dali.
In the context a CDistributedFile is being used for this PR, to faciliate/re-use many of the implementation of IDistributeFile methods, the instance is unattached and should not reach out to Dali.